exchange/bidder.go

```diff
@@ -87,6 +87,8 @@ type pbsOrtbSeatBid struct {
 	// if len(bids) > 0, this will become response.seatbid[i].ext.{bidder} on the final OpenRTB response.
 	// if len(bids) == 0, this will be ignored because the OpenRTB spec doesn't allow a SeatBid with 0 Bids.
 	ext json.RawMessage
+
+	warnings map[openrtb_ext.BidderName][]openrtb_ext.ExtBidderMessage
```

Comment:
Is the idea to create a new pathway for reporting on warnings? Right now, warnings are error types with a lower severity code. This looks to change that approach and I'm unsure how that will fit in with our current error handling infrastructure.